<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWidget</class>
 <widget class="QWidget" name="EditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit and/or Add New songs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="softprojector.qrc">
    <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Songbook:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="songbook_label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="song_number_lineEdit">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Title: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTitle"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Words by:  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditWordsBy"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Music by: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditMusicBy"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Key: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditKey"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Category: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboBoxCategory">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Библейские истории</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Благовествование</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Бог</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Божья любовь и величие</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Время</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Воскресение Христово</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Второе пришествие Христа и суд</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Для детей и семейные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Духовная борьба и победа</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Для новообращённых</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Жатвенные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Иисус Христос</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Любовь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Мама</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Молитва</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Молитвенные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Молодёжные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На бракосочетание</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На закате</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Наш Друг и Спаситель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На крещение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На новый год</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На nогребение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На рассвете</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На рукоположение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">На хлебопреломление</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Небесные обители</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Наставление и самоиспытание</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">О Духе Святом, освящение, через Духа Святого</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">О Церкви</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Перед началом собрания</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Последнее время</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Практическая жизнь с Богом</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">При заключении собрания</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Приветственные и nрощальные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Призыв к труду</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Призыв к покаянию, о покаянии</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Путь веры</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Радость спасения во Христе</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Разные христианские праздники</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Рождество христово</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Решительность и верность</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Руки Христа</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Следование</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Слово Божье</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Спасение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Страдание и смерть Христа</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Уверенность в Господе</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Утешение больным, страждущим узникам</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Утешение и ободрение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Хвала и благодарение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Христианская радость</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="aling_groupBox">
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Text Alingment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QRadioButton" name="left_radioButton">
        <property name="text">
         <string>Left</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="center_radioButton">
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rihgt_radioButton">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="font_gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="font_textbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="font_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="font_default_button">
       <property name="text">
        <string>Use default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="font_label">
       <property name="text">
        <string>Text font:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="wall_gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="wall_label">
       <property name="text">
        <string>Background:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="wall_textbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="wall_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="wall_default_button">
       <property name="text">
        <string>Use default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QTextEdit" name="textEditSong"/>
   </item>
   <item row="8" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>168</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Q</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
